modules/Router.js

```diff
@@ -62,6 +62,8 @@ const Router = React.createClass({
       if (error) {
         this.handleError(error)
       } else {
+        router.location = state.location
+        router.params = state.params
```

Comment:
I thought this was somehow the leg work to get eg `location` in redux as well, if it isn't totally my bad 😁 and feel free to ignore. ([context](https://twitter.com/dan_abramov/status/729712661856915457))